cv_lib/cv_lib/utils.py

```diff
@@ -1,6 +1,45 @@
+  
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
 import os
 import logging
+from PIL import Image
+import numpy as np
+from matplotlib import pyplot as plt
+
+def normalize(array):
+    """
+    Normalizes a segmentation mask array to be in [0,1] range
+    for use with PIL.Image
+    """
+    min = array.min()
+    return (array - min) / (array.max() - min)
+
+def mask_to_disk(mask, fname, cmap_name="Paired"):
+    """
+    write segmentation mask to disk using a particular colormap
+    """
+    cmap = plt.get_cmap(cmap_name)
+    Image.fromarray(cmap(normalize(mask), bytes=True)).save(fname)
```

Comment:
I think `normalize()` should only be used for seismic images (continuous), and not the masks (discrete) right?

Reply:
and in that case, we would not need cmap

Reply:
discussed - to the best of my knowledge camp works on [0,1] image, so we need to convert to that range